run.py

```diff
@@ -53,6 +54,31 @@ def on_app_mention(event_data):
     channel = event_data["event"]["channel"]
     blocks = event_data['event']['blocks']
     authed_users = event_data['authed_users']
+    client_msg_id = event_data['event']['client_msg_id']
+
+    print(f'event_data: {event_data}')
+    print(f'channel: {channel}')
+    print(f'blocks: {blocks}')
+    print(f'authed_users: {authed_users}')
+    print(f'client_msg_id: {client_msg_id}')
+
+    with Database() as _db, _db.conn.cursor() as cursor:
+        cursor.execute(
+            'SELECT client_msg_id FROM slack_client_msg_id WHERE client_msg_id = %s LIMIT 1',
+            (client_msg_id,))
+
+        if cursor.fetchone():
+            print('skip')
+            return
+
+        cursor.execute(
+            "DELETE FROM slack_client_msg_id WHERE '00:10:00' < CURRENT_TIMESTAMP - created_at",
```

Comment:
```suggestion
            "DELETE FROM slack_client_msg_id WHERE created_at < CURRENT_TIMESTAMP - '00:10:00'",
```

の方がDB的には優しい。INDEX貼ってあるなら尚更。

Reply:
ところで `cursor.fetchone()` する前に消さないと、10分過ぎてるけどデータとしてはある、みたいなのでreturnされるけどそれはよろし？

Reply:
件数が1,000件行かなければ良いので、INSERT時に消せば良いという認識でいる。

Reply:
クエリはこのままいく？

Reply:
あ、すまん、一番上のコメント見てなかった。
反映させます。